tests/Benchmarks/System.Binary.Base64/Stiching.cs

```diff
@@ -0,0 +1,73 @@
+﻿using BenchmarkDotNet.Attributes;
+using System;
+using System.Binary.Base64Experimental;
+using Base64Decoder = System.Buffers.Text.Base64;
+
+namespace Benchmarks.System.Binary.Base64.Benchmarks
+{
+    public class Stiching
+    {
+        [Params(1000, 5000, 10000, 20000, 50000)]
```

Comment:
nit: use underscores to make the values easier to read.

For example, `50000` => `50_000`